package.json

```diff
@@ -140,7 +140,7 @@
         "firebase-tools": "6.1.1",
         "glob": "^7.1.3",
         "husky": "^2.2.0",
-        "jest": "^24.9.0",
+        "jest": "^26.1.0",
```

Comment:
We need to be careful about this updates since there may be Breakin Changes, I agree this needs to be done, but it should be another task where release notes are reviewed in order to not break our library.